### PR TITLE
Opsmetricsv2: send counters for API calls

### PR DESCRIPTION
**Pull request by brendanobra**
## What

Refactoring operational metrics to be standalone + adding counters for firebolt api calls

## Why
refactoring is to simplify things a bit, and counters are to provide better (lower cardinality) ops metrics, leaning into rust' stong typing a bit more to gain more advantage over high cardinality on backend : status has been forced into a defined type (not just an aribtrary struct), and the string that was formerly holding status has been replaced with the struct and types to provide conversion to the status field.

## How
added counters
## Test

How has this been tested? How can a reviewer test it?

## Checklist

- [ ] I have self-reviewed this PR
- [ ] I have added tests that prove the feature works or the fix is effective

### CODE REVIEW

File: core/sdk/src/api/observability/operational_metrics.rs [hunk 1]
```diff
@@ -1,9 +1,220 @@
...
 };
 
+/*
+Operational Metrics. These are metrics that are not directly related to the user's behavior. They are
+more related to the operation of the platform itself. These metrics are not sent to the BI system, and
+are only used for operational/performance measurement - timers, counters, etc -all of low cardinality.
+*/
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+pub struct Counter {
```
Comment (pahearn73) [inline core/sdk/src/api/observability/operational_metrics.rs:17]: This (and Timer, etc.) seem almost the same as those defined in `core/sdk/src/api/observability/mod.rs`?

File: core/sdk/src/api/firebolt/fb_metrics.rs [hunk 2]
```diff
@@ -827,6 +625,18 @@ impl MetricsContext {
             authenticated: None,
         }
     }
+
+    pub fn get_fw_version(&self) -> String {
```
Comment (satlead) [inline core/sdk/src/api/firebolt/fb_metrics.rs:629]: Seems RDK Release specific perhaps extension might be a better fit

File: core/sdk/src/api/observability/operational_metrics.rs (hunk 1 above)
Comment (brendanobra) [reply core/sdk/src/api/observability/operational_metrics.rs:17]: thanks , will clean up

File: core/sdk/src/api/firebolt/fb_metrics.rs (hunk 2 above)
Comment (brendanobra) [reply core/sdk/src/api/firebolt/fb_metrics.rs:629]: do u mean the name of the function is too RDK release specific, or that this function should be in the extension?

File: core/sdk/src/api/observability/operational_metrics.rs (hunk 1 above)
Comment (brendanobra) [reply core/sdk/src/api/observability/operational_metrics.rs]: cleaned

File: core/sdk/src/api/firebolt/fb_metrics.rs (hunk 2 above)
Comment (satlead) [reply core/sdk/src/api/firebolt/fb_metrics.rs:629]: yeah terms VBN and PROD are naming conventions used by RDK Release. Feel they would change in the future impacting our Ripple core. Having them in extension before they are exported seems a better strategy

**Review by satlead — APPROVED** [review]
RDK release specific code needs to be cleaned up to an extension in a future change

**Review by pahearn73 — APPROVED** [review]

**Pull request closed without merging**